auto-elr/build.gradle

```diff
@@ -38,13 +38,15 @@ dependencies {
   implementation project(":runners:portability:java")
 
   implementation project(":sdks:java:core")
+  implementation project(":sdks:java:extensions:google-cloud-platform-core")
```

Comment:
is `com.linkedin.beam:beam-sdks-java-io-google-cloud-platform` also required as per error msg?

Reply:
Yes.

```
            > Could not GET 'https://repo.spring.io/plugins-release/com/linkedin/beam/beam-sdks-java-io-google-cloud-platform/200.45.13.101/beam-sdks-java-io-google-cloud-platform-200.45.13.101.pom'. Received status code 401 from server: 

```

Reply:
Good catch, thanks!